extensions/coroutines-extensions/src/commonTest/kotlin/com/squareup/sqldelight/runtime/coroutines/QueryAsFlowTest.kt

```diff
@@ -94,7 +86,7 @@ class QueryAsFlowTest {
     }
   }
 
-  @Test fun queryCanBeCollectedMoreThanOnce() = runTest {
+  @Test fun queryCanBeCollectedMoreThanOnce() = runTest(cleanupAfter = 1.seconds) { db ->
```

Comment:
instead of having `cleanupAfter` can this test be rewritten to spin while the coroutines haven't finished yet?

```
val flow = db.createQuery(TABLE_EMPLOYEE, "$SELECT_EMPLOYEES WHERE $USERNAME = 'john'", MAPPER)
        .asFlow()
        .mapToOneNotNull()

val timesCanceled = AtomicInt()
    val employee = Employee("john", "John Johnson")
    repeat(5) {
      launch {
        flow.test {
          assertEquals(employee, expectItem())
          cancel()
          timesCanceled.increment()
        }
      }
    }
    db.employee(employee)
    while (timesCanceled.get() != 5) {
      // is yield a thing in coroutines?
    }
```